### PR TITLE
Optimize GDM + Small fixes

### PR DESCRIPTION
**Pull request by bbfsdev**

### CODE REVIEW

File: src/apps/VirtualApp/VirtualClient.js [hunk 1]
```diff
@@ -600,6 +600,7 @@ class VirtualClient extends Component {
       },
       onremotestream: (stream) => {
         // The publisher stream is sendonly, we don't expect anything here
+        Janus.log('Send only publisher stream, if this happends, it is not expected, stream:', stream);
```
Comment (edoshor) [inline src/apps/VirtualApp/VirtualClient.js:603]: Escalate to warn level

File: src/apps/VirtualApp/VirtualClient.js [hunk 2]
```diff
@@ -649,13 +650,12 @@ class VirtualClient extends Component {
 
   onMessage = (videoroom, msg, jsep) => {
     const {t} = this.props;
-    Janus.log(' ::: Got a message (publisher) :::');
-    Janus.log(msg);
-    let event = msg['videoroom'];
+    Janus.log(`::: Got a message (publisher) ::: ${JSON.stringify(msg)}`);
```
Comment (edoshor) [inline src/apps/VirtualApp/VirtualClient.js:653]: no reason to stringify explicitly. Try `Janus.log('::: Got a message (publisher) :::', msg);`

**Review by edoshor — CHANGES_REQUESTED** [review]

File: src/apps/VirtualApp/VirtualClient.js (hunk 1 above)
Comment (bbfsdev) [reply src/apps/VirtualApp/VirtualClient.js:603]: Done.

File: src/apps/VirtualApp/VirtualClient.js (hunk 2 above)
Comment (bbfsdev) [reply src/apps/VirtualApp/VirtualClient.js:653]: In some cases I want to have all details in Sentry, which is not possible due to the fact that Sentry will truncate after printing one level of details... 

See "[object MessageEvent]"


console | Received message on data channel: [object MessageEvent]
-- | --
logger | console
arguments | ["Received message on data channel:",{"isTrusted":"[object MessageEvent]","type":"message","currentTarget":"[object RTCDataChannel]","target":"[object RTCDataChannel]"}]

**Review by edoshor — APPROVED** [review]

**Pull request merged**